iotedgedev/deploymentmanifest.py

```diff
@@ -147,3 +186,74 @@ def _get_module_content(self):
             return self.json["moduleContent"]
         else:
             raise KeyError("modulesContent")
+
+    def _validate_deployment_manifest_schema(self):
+        validation_success = True
+        try:
+            self.output.info("Validating schema of deployment manifest.")
+            deployment_schema = json.loads(urlopen(Constants.deployment_manifest_schema_url).read())
+            validator_class = jsonschema.validators.validator_for(deployment_schema)
```

Comment:
cannot use jsonschema.validate() method?

Reply:
If there're multiple schema errors, jsonschema.validate() only returns 1 error. So user could not know remaining errors and fix them.
The validator way could give us all schema errors, so I implemented in this way.